### DISCUSSION TITLE
Does $os.removeAll() work?

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#3895] [Q&A] [answered]
I am running `$os.removeAll(path)` on a path I know exists, but it's not deleting any children. This is on my local machine and it's not a permission issue:

```bash
mkdir -p /path/to/test
```

```js
$os.removeAll(`/path/to/test`)
```

Expected: `test` directory is deleted

Actual: `test` directory remains

**Comment by ganigeorgiev** [comment, marked as answer]
`$os.removeAll()` directly calls the Go `os.RemoveAll()` function.
 
I've just tested it locally with both file and directory as path and it works fine for me (aka. in both cases the file/directory is deleted).

Double check that the correct path is provided.
You could also to try to wrap the call with `try/catch` to inspect if there is any error thrown.
